classes/person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rentals(date, book, _person)
+    @rentals << Rental.new(date, book, self)
+  end
+
```

Comment:
- [ ] Since you are in the Person class already, then the add_rental method should optimized like the following:
```suggestion
  def add_rentals(date, book)
    @rentals << Rental.new(date, book, self)
  end

```

Reply:
@descholar-ceo Thank you for the review and the alternate implementation. I will update the code accordingly.